ResearchUXFactory/SBABaseSurveyFactory.swift

```diff
@@ -58,16 +53,83 @@ open class SBASurveyFactory : NSObject, SBASharedInfoController {
     
     public convenience init(dictionary: NSDictionary) {
         self.init()
+        
+        // Load the sections
+        var previousSectionType: SBAConsentSectionType?
```

Comment:
These were pulled into the base class b/c I couldn't support multiple inheritance in the default override class defined in BridgeAppSDK. I can update the comments.